flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -1666,26 +1755,32 @@
                             <ANIM_NAME>KNOB_OVHD_ADIRS_3</ANIM_NAME>
                         </UseTemplate>
 
-                        <UseTemplate Name="FBW_AIRBUS_Push_Dummy_Template">
```

Comment:
I like that you are replacing `FBW_AIRBUS_Push_Dummy_Template`. At some point we should convert the rest that is not yet converted.